.bootstraprc

```diff
@@ -1,18 +1,89 @@
-{
-  "bootstrapVersion": 3,
-  "styleLoaders": ["style", "css", "sass"],
-  "extractStyles": true,
-  "styles": {
-    "mixins": true,
-    "glyphicons": true,
-    "grid": true,
-    "forms": true,
-    "input-groups": true,
-    "responsive-utilities":true,
-    "panels": true,
-    "type": true
-  },
-
-  "scripts": false,
-  "styleNamespace": ".tw-bs"
-}
\ No newline at end of file
+---
+# Output debugging info
+loglevel: debug
+
+# Major version of Bootstrap: 3 or 4
+bootstrapVersion: 4
+
+# Webpack loaders, order matters
+styleLoaders:
+  - style-loader
+  - css-loader
+  - postcss-loader
+  - sass-loader
+
+# Extract styles to stand-alone css file
+# Different settings for different environments can be used,
+# It depends on value of NODE_ENV environment variable
+# This param can also be set in webpack config:
+#   entry: 'bootstrap-loader/extractStyles'
+extractStyles: false
+styleNamespace: .tw-bs
+# env:
+#   development:
+#     extractStyles: false
+#   production:
+#     extractStyles: true
+
+
+# Customize Bootstrap variables that get imported before the original Bootstrap variables.
+# Thus, derived Bootstrap variables can depend on values from here.
+# See the Bootstrap _variables.scss file for examples of derived Bootstrap variables.
+#
+# preBootstrapCustomizations: ./path/to/bootstrap/pre-customizations.scss
+
+
+# This gets loaded after bootstrap/variables is loaded
+# Thus, you may customize Bootstrap variables
+# based on the values established in the Bootstrap _variables.scss file
+#
+# bootstrapCustomizations: ./path/to/bootstrap/customizations.scss
+
+
+# Import your custom styles here
+# Usually this endpoint-file contains list of @imports of your application styles
+#
+# appStyles: ./path/to/your/app/styles/endpoint.scss
+
+
+### Bootstrap styles
+styles:
+
+  # Mixins
+  mixins: true
+
+  # Reset and dependencies
+  print: true
+
+  # Core CSS
+  buttons: true
+  code: true
+  forms: true
+  grid: true
+  images: true
+  reboot: true
+  tables: true
+  type: true
+
+  # Components
+  alert: true
+  badge: true
+  breadcrumb: true
+  button-group: true
+  card: true
+  close: true
+  custom-forms: true
+  dropdown: true
+  input-group: true
+  jumbotron: true
+  list-group: true
+  media: true
+  nav: true
+  navbar: true
+  pagination: true
+  progress: true
+  transitions: true
+
+  # Components w/ JavaScript
+  carousel: true
+  modal: true
```

Comment:
Did we have to change from JSON to YAML here? I don't really have a preference for one or the other but I'd prefer to limit the number of changes as much as possible.

Reply:
I believe I just grabbed the defaults from
https://github.com/shakacode/bootstrap-loader/blob/master/.bootstraprc-4-default